docs/nitpick-exceptions

```diff
@@ -134,3 +134,17 @@ py:class ModuleType
 py:class NDArray
 # locally defined type variable for ndarray dtype
 py:class DT
+# astropy.units
+py:obj PhysicalType
+py:obj UnitBase
+py:class Unit
+py:obj Unit
+py:obj PrefixUnit
+py:obj StructuredUnit
+py:obj UnrecognizedUnit
+py:obj CompositeUnit
+py:class Equivalency
+py:obj Equivalency
+py:class Quantity
+py:func astropy.units.Quantity
+py:obj Quantity
```

Comment:
I'd prefer if this was kept alphabetically sorted.

Reply:
Sure! When I finally get it all working I'll push one more commit alphabetizing. Do you mean alphabetize by the `py` flag (`py:class` before `py:obj`) or by the object (`Equivalency` before `Quantity`).

Reply:
I'd say `py:class` before `py:obj`.

Reply:
Done.